frontend/ui/wikipedia.lua

```diff
@@ -768,9 +768,9 @@ time, abbr, sup {
 
     -- Done with adding files
     if cancelled then
-        progress_callback(_("Cleaning up..."))
+        progress_callback(_("Cleaning up…"))
```

Comment:
Actually it's fine in the program itself; it's just GitHub that displays it like this. Sorry, I hadn't checked before. Although it's still my preferred solution to not have any weirdness like that in an actual file because then it causes issues like these in text editors and GitHub.